caribic/src/start.rs

```diff
@@ -517,7 +550,8 @@ pub async fn prepare_osmosis(osmosis_dir: &Path) -> Result<(), Box<dyn std::erro
     match copy_osmosis_config_files(osmosis_dir) {
         Ok(_) => {
             verbose("✅ Osmosis configuration files copied successfully");
-            remove_previous_chain_data()?;
+            // This should not be required as each time it's restarted it wipes the .osmosisd-local data
+            // remove_previous_chain_data()?;
```

Comment:
@fabianbormann this cleanup doesn't seem to be necessary. You can check the README.md in the `localosmosis` folder

```
```bash
make localnet-init
```

The command:

- Builds a local docker image with the latest changes
- Cleans the `$HOME/.osmosisd-local` folder

```

Reply:
Ah ok makes sense. It's just a re-write of the bash script. I assume we can just remove it ;)